<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMacWid</class>
 <widget class="QWidget" name="CreateMacWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>MAC地址生成</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>客户名称：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设备SN：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="typeBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>IP-PDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MPDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RPDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ZPDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ATS</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>设备类型：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="snEdit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>210</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="createBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
     <widget class="QPushButton" name="revokeBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>撤销</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="macEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设备MAC地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="copyBtn">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>复制</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="startEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>340</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>起始MAC地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="endEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>370</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>结束MAC地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="countEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>300</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>300</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>剩余MAC数量：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
